mne_connectivity/viz/_3d.py

```diff
@@ -37,9 +43,12 @@ def plot_sensors_connectivity(
     n_con : int
         Number of strongest connections shown. By default 20.
     cmap : str | instance of matplotlib.colors.Colormap
-        Colormap for coloring connections by strength. If :class:`str`, must
-        be a valid Matplotlib colormap (i.e. a valid key of
-        ``matplotlib.colormaps``). Default is ``"RdBu"``.
+        Colormap for coloring connections by strength. If :class:`str`, must be a valid
+        Matplotlib colormap (i.e. a valid key of `matplotlib.colormaps`). Default is
+        ``"RdBu"``.
+    min_distance : float
+        The minimum distance required between two sensors to plot a connection between
+        them, in meters. Default is 0.05 (i.e. 5 cm).
```

Comment:
Is sticking with the default unit of metres okay, or would it be more intuitive to use something smaller in this case, e.g. cm?

Reply:
I even used the American spelling to be nice